style.css

```diff
@@ -173,10 +216,6 @@ ul {
   display: flex;
 }
 
-a:last-child {
-  margin-right: 0;
-}
-
 @media (min-width: 480px) {
```

Comment:
- [OPTIONAL] Great job so far! 👏🏻  however according to the project requirement, 🤔  on the desktop size,  the closed button, and the menu mobile page must not be visible. Kindly correct this. You can use your `@media screen and (min-width: 480px) {`  to display none of your navigation mobile menu. 😃 

- Your design:

![18](https://user-images.githubusercontent.com/79058364/188521348-c0f9b3e1-8130-419f-a088-42d2b6eaf62e.png)